src/core/job.c

```diff
@@ -714,11 +715,16 @@ static void job_print_status_message(Unit *u, JobType t, JobResult result) {
         if (!format)
                 return;
 
+        if (log_get_show_color())
+                status = strjoina(statuses[result][0], statuses[result][1], ANSI_NORMAL);
```

Comment:
Can we please make this a proper struct with proper member names that describe what they are doing?